index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Gaia Media Controls Demo</title>
+  <meta name="description" content="Demo app for Gaia media controls component">
+  <script src="bower_components/gaia-icons/gaia-icons.js"></script>
```

Comment:
I see you do include gaia-icons explicitly here, so I don't think you should have the font-face and data-icon stuff in your own css.  But maybe I'm just confused about this.